docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -398,6 +398,76 @@ Finally
           as the block-direct policies may yield better performance.
 
 
+ GPU Policies for SYCL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ ======================================== ============= ========================
+ SYCL Execution Policies                  Works with    Brief description
+ ======================================== ============= ========================
+ sycl_exec<WORK_GROUP_SIZE>               forall,       Execute loop iterations
+                                                        in a GPU kernel launched
+                                                        with given work group
+                                                        size.
+ sycl_global_1<WORK_GROUP_SIZE>           kernel (For)  Map loop iterates
```

Comment:
@homerdin it looks like the sycl exec policies have been changed from sycl_*_{1,2,3} to sycl_*_{0,1,2}. Please update the polcy documentation here.  Thank you.